README.md

```diff
@@ -213,7 +215,20 @@ launchctl list | grep mctomqtt           # Check status
 tail -f ~/Library/Logs/mctomqtt.log      # View logs
 ```
 
-### Manual Execution
+### 2. Docker Container
+Isolated containerized deployment with automatic restarts.
+
```

Comment:
I think this needs an example `docker run` command. Additionally, I think the docker "operational" commands below are well documented enough in the docker docs, they probably don't need to be here.